packages/components/src/styles/mixins/_interactive-dark-theme.scss

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+ @use "../mixins/focus-ring" as *;
+
+ // TODO: Deprecate SideNav specific dark theme tokens in favor of general dark theme tokens
+
+// Used to apply dark theme to interactive elements such as Button & Dropdown
+@mixin hds-interactive-dark-theme($add-visible-border: true, $is-within-side-nav: false) {
+  @if $is-within-side-nav {
```

Comment:
It's not _that_ bad, at least it's hidden all in one place

Reply:
Thanks, to me I think this is ok for phase 1 & could be refined/replaced as needed in upcoming phases.